protocol/messenger_communities.go

```diff
@@ -1314,13 +1314,13 @@ func (m *Messenger) passStoredCommunityInfoToSignalHandler(communityID string) {
 		return
 	}
 
-	//if there is no info helpful for client, we don't post it
-	if community.Name() == "" && community.DescriptionText() == "" && community.MembersCount() == 0 {
+	if err != nil {
+		m.logger.Warn("cant get community and pass it to signal handler", zap.Error(err))
 		return
 	}
 
-	if err != nil {
-		m.logger.Warn("cant get community and pass it to signal handler", zap.Error(err))
+	//if there is no info helpful for client, we don't post it
+	if community.Name() == "" && community.DescriptionText() == "" && community.MembersCount() == 0 {
```

Comment:
Change is unrelated to PR, but makes sure that we check error before accessing community.